src/sentry/integrations/jira_server/actions/create_ticket.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import annotations
+
+from typing import Any
+
+from sentry.integrations.jira_server.actions.form import JiraServerNotifyServiceForm
+from sentry.rules.actions import TicketEventAction
+from sentry.services.hybrid_cloud.integration import RpcIntegration
+from sentry.utils.http import absolute_uri
+
+
+class JiraServerCreateTicketAction(TicketEventAction):
+    id = "sentry.integrations.jira_server.notify_action.JiraServerCreateTicketAction"
+    label = "Create a Jira issue in {integration} with these "
```

Comment:
Should the label and ticket type say "Jira Server" instead? Or maybe that's a little redundant when it's filled in so it'd say" Create a Jira Server issues in Jira Server with these "

Reply:
`{integration}` actually ends up being the integration instance name, not the integration itself so it should be Jira Server, I'll update it.
<img width="635" alt="Screenshot 2023-10-16 at 1 57 38 PM" src="https://github.com/getsentry/sentry/assets/67301797/5f913f03-b641-4577-b5ef-1515c76583ae">